tools/ci/tc/tasks/test.yml

```diff
@@ -0,0 +1,339 @@
+    schedulerId: taskcluster-github
+    deadline: "24 hours"
+    image: harjgam/web-platform-tests:0.33
+    maxRunTime: 7200
+    artifacts:
+      public/results:
+        path: /home/test/artifacts
+        type: directory
+
+  wpt-testharness:
+    chunks: 12
+    vars:
+      test-type: testharness
+
+  wpt-reftest:
+    chunks: 6
+    vars:
+      test-type: reftest
+
+  wpt-wdspec:
+    chunks: 1
+    vars:
+      test-type: wdspec
+
+  run-options:
+    options:
+      xvfb: true
+      oom-killer: true
+      hosts: true
+      install-certificates: true
+
+  wpt-run:
+    name: wpt-${vars.browser}-${vars.channel}-${vars.suite}-chunk-${chunks.id}
+    options:
+      browser:
+        - ${vars.browser}
+      channel: ${vars.channel}
+    command: >-
+      ./tools/ci/taskcluster-run.py
+      ${vars.browser}
+      --
+      --channel=${vars.channel}
+      --log-wptreport=../artifacts/wpt_report.json
+      --log-wptscreenshot=../artifacts/wpt_screenshot.txt
+      --no-fail-on-unexpected
+      --this-chunk=${chunks.id}
+      --total-chunks=${chunks.total}
+
+  trigger-master:
+    trigger:
+      branch:
+        - master
+
+  trigger-push:
+    trigger:
+      branch:
+        - triggers/${vars.browser}_${vars.channel}
+
+  trigger-daily:
+    trigger:
+      branch:
+        - epochs/daily
+
+  trigger-weekly:
+    trigger:
+      branch:
+        - epochs/weekly
+
+  trigger-pr:
+    trigger:
+      pull-request:
+
+  browser-firefox:
+    require:
+      - download-firefox-${vars.channel}
+
+  browser-chrome: {}
+
+  tox-python2:
+    env:
+      TOXENV: py27
+      PY_COLORS: 0
+
+  tox-python3:
+    env:
+      TOXENV: py36
+      PY_COLORS: 0
+    install:
+      - python3-pip
+
+tasks:
+  # Run full suites on push
+  - $map:
+      for:
+        - vars:
+            suite: testharness
+        - vars:
+            suite: reftest
+        - vars:
+            suite: wdspec
+      do:
+        $map:
+          for:
+            - vars:
+                browser: firefox
+                channel: nightly
+              use:
+                - trigger-master
+                - trigger-push
+            - vars:
+                browser: firefox
+                channel: beta
+              use:
+                - trigger-weekly
+                - trigger-push
+            - vars:
+                browser: firefox
+                channel: stable
+              use:
+                - trigger-daily
+                - trigger-push
+            - vars:
+                browser: chrome
+                channel: dev
+              use:
+                - trigger-master
+                - trigger-push
+            - vars:
+                browser: chrome
+                channel: beta
+              use:
+                - trigger-weekly
+                - trigger-push
+            - vars:
+                browser: chrome
+                channel: stable
+              use:
+                - trigger-daily
```

Comment:
The naming of these two isn't very clear in context, especially here my first read was that it'd trigger daily and on every push. Maybe "trigger-manual" instead of "trigger-push"? Or "trigger-trigger" (jk)